suite-native/module-accounts-import/src/components/DevXpub.tsx

```diff
@@ -18,8 +17,8 @@ const devXpubs: Partial<Record<NetworkSymbol, string>> = {
     btg: 'ypub6WVGAPrpnuTJN8AULEnWNNpvFR5Xcckn66fp8sL4FgEy5Z4oig1CbwySvj8v4iFVSANcHJVWzK211Bb8u71bCTVW9vY1PbAn59KmCzq99YW',
     dgb: 'ypub6X7pNV6ouYFiDGHjxCtbnV9EaCdq5uyVysMbR5Q79LHa3SWV93J7ubun37EJhfFQqsSGQBfz3UrAzNtYNhb5JsoPJbNKvbF9wKxBjgxfXkH',
     zec: 'xpub6CQdEahwhKRSn9BFc7oWpzNoeqG2ygv3xdofyk7He93NMjvDpGvcQ2o4dZfBNXpqzKydaHp5rhXRT3zYhRYJAErXxarH37f9hgRZ6UPiqfg',
-    eth: '0x73d0385F4d8E00C5e6504C6030F47BF6212736A8', // Public key, not xpub
-    ada: 'd507c8f866691bd96e131334c355188b1a1d0b2fa0ab11545075aab332d77d9eb19657ad13ee581b56b0f8d744d66ca356b93d42fe176b3de007d53e9c4c4e7a',
+    eth: '0x62270860B9a5337e46bE8563c512c9137AFa0384', // Public key, not xpub
+    ada: 'addr1qxwy5p2kxd8xeclxxgx4lmhuae0knu762etrjgg8772gepp6sjecwjdttznf0hqxwc0suz8axtj3yjwzmnl99r9le56sc6hyez',
```

Comment:
Have you tried the new key for ADA? I believe it is not a key, but a public address.